crates/biome_js_analyze/tests/specs/nursery/useGoogleFontPreconnect/invalid.jsx

```diff
@@ -0,0 +1,4 @@
+<>
+	<link href="https://fonts.gstatic.com"/>
+	<link rel="preload" href="https://fonts.gstatic.com"/>
```

Comment:
nit: you might want to add *at least* one case where the attributes are on multiple lines:

```jsx
<link 
   rel="preload" 
   href="https://fonts.gstatic.com"
/>
```

Most probably, the current implementation of the code action doesn't handle properly this case